init-tools.sh

```diff
@@ -22,47 +22,59 @@ case $OSName in
 
     Linux)
         OS=Linux
-        __DOTNET_PKG=dotnet-ubuntu-x64
+        source /etc/os-release
+        if [ $ID = "centos" ]; then
+            __DOTNET_PKG=dotnet-centos-x64
+        elif [ $ID = "ubuntu" -o $ID = "debian" ]; then
+            __DOTNET_PKG=dotnet-ubuntu-x64
+        else
+            echo "Unsupported ID '$ID' detected. Downloading ubuntu-x64 tools."
+            __DOTNET_PKG=dotnet-ubuntu-x64
+        fi
         ;;
 
     *)
-        echo "Unsupported OS $OSName detected. Downloading ubuntu-x64 tools"
+        echo "Unsupported OS '$OSName' detected. Downloading ubuntu-x64 tools."
         OS=Linux
         __DOTNET_PKG=dotnet-ubuntu-x64
         ;;
 esac
 
 if [ ! -e $__PROJECT_JSON_FILE ]; then
- if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
+    if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
 
- if [ ! -e $__DOTNET_PATH ]; then
-    # curl has HTTPS CA trust-issues less often than wget, so lets try that first.
-    which curl > /dev/null 2> /dev/null
-    if [ $? -ne 0 ]; then
-      mkdir -p "$__DOTNET_PATH"
-      wget -q -O $__DOTNET_PATH/dotnet.tar https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz
-    else
-      curl -sSL --create-dirs -o $__DOTNET_PATH/dotnet.tar https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz
-    fi
-    cd $__DOTNET_PATH
-    tar -xf $__DOTNET_PATH/dotnet.tar
-    if [ -n "$BUILDTOOLS_OVERRIDE_RUNTIME" ]; then
-        find $__DOTNET_PATH -name *.ni.* | xargs rm 2>/dev/null
-        cp -R $BUILDTOOLS_OVERRIDE_RUNTIME/* $__DOTNET_PATH/bin
-        cp -R $BUILDTOOLS_OVERRIDE_RUNTIME/* $__DOTNET_PATH/bin/dnx
-        cp -R $BUILDTOOLS_OVERRIDE_RUNTIME/* $__DOTNET_PATH/runtime/coreclr
-    fi
+    if [ ! -e $__DOTNET_PATH ]; then
+        echo "Installing dotnet cli..."
+        # curl has HTTPS CA trust-issues less often than wget, so lets try that first.
+        which curl > /dev/null 2> /dev/null
+        if [ $? -ne 0 ]; then
+            mkdir -p "$__DOTNET_PATH"
+            wget -q -O $__DOTNET_PATH/dotnet.tar https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz
+        else
+            curl -sSL --create-dirs -o $__DOTNET_PATH/dotnet.tar https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz
+        fi
+        cd $__DOTNET_PATH
+        tar -xf $__DOTNET_PATH/dotnet.tar
```

Comment:
I had noticed that earlier and was planning to fix it, but it was actually fixed in https://github.com/dotnet/corefx/commit/c180cc0ef7adfde965514e0bce85e09668b09a9d.